docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-score-constraints-guided-rules-proc.adoc

```diff
@@ -0,0 +1,36 @@
+[id='optimizer-score-constraints-guided-rules-proc']
+= Define score constraints using the guided rules designer
+
+
+To solve an optimization problem, you need to define score constraints that evaluate your solution.
+{PLANNER} integrates with the guided rules designer and provides score modifiers, which are used by the engine during the solving process.
+
+Score modifiers can be accessed in the action selector (+) of the *THEN* (right-hand side) section of a rule, provided the planning solution is defined within the project.
+
+Use the following procedure to create a new score constraint using the guided rules designer.
+
+
+.Prerequisite
+
+You have defined a https://docs.jboss.org/optaplanner/release/latest/optaplanner-docs/html_single/index.html#solutionClass[Planning Solution]. 
+// NOTE FOR SME REVIEWER: where/how is this defined in the Employee Rostering example? 
```

Comment:
By creating a class in the domain editor, click on the right on the optaplanner icon and clicking the radio button "planning solution"

Reply:
Updated the prerequisite.